app/assets/src/components/views/discovery/mapping/DiscoveryMap.jsx

```diff
@@ -88,33 +114,49 @@ class DiscoveryMap extends React.Component {
   };
 
   renderMarker = locationInfo => {
-    const { currentTab, previewedLocationId } = this.props;
+    const { currentTab, mapLevel, previewedLocationId } = this.props;
     const { viewport } = this.state;
+
     const id = locationInfo.id;
     const name = locationInfo.name;
     const lat = parseFloat(locationInfo.lat);
     const lng = parseFloat(locationInfo.lng);
+    const geoLevel = locationInfo.geo_level;
+
     const idsField = currentTab === "samples" ? "sample_ids" : "project_ids";
     if (!locationInfo[idsField]) return;
+
     const pointCount = locationInfo[idsField].length;
-    const minSize = 10;
+    // Sizing parameters determined via eyeballing.
+    const minSize = 14;
```

Comment:
Can we move all the magic eyeballed numbers to props?

Reply:
👍